app/models/accompaniment.rb

```diff
@@ -3,6 +3,7 @@ class Accompaniment < ApplicationRecord
   belongs_to :activity
 
   validates :activity_id, :user_id, presence: true
+  validate :limit_for_family_court
```

Comment:
In QA, am seeing 'No template found for AccompanimentsController#create, rendering head :no_content' when I click 'Attend' for a family court activity that has met the cap because we aren't rendering anything on the 'save' fail.

Reply:
ah! ack - sorry about that.